pkg/ssh/client/ssh.go

```diff
@@ -133,8 +133,11 @@ func importKey(ctx context.Context, keyName, fingerprint string, keyContent *str
 	if existing, err := findKeyInEC2(ctx, ec2API, keyName); err != nil {
 		return err
 	} else if existing != nil {
-		if *existing.KeyFingerprint != fingerprint {
-			return fmt.Errorf("SSH public key %s already exists, but fingerprints don't match (exected: %q, got: %q)", keyName, fingerprint, *existing.KeyFingerprint)
+		// AWS adds an extra `=` to ed25519 sha256 encodings. Normally, that's not there.
```

Comment:
> AWS adds an extra `=` to ed25519 sha256 encodings.

Is there an "and" missing between the two types? The sentence is a bit confusing 🤔